src/components/common/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+
+interface PropsTypes {
+  text: string;
+  onClick: () => void;
+  color: string;
+}
+
+const Container = styled.button`
+  background-color: ${(props) => props.color};
+  border: none;
+  color: var(--fontColor);
+`;
+
+export default function Button({ text, onClick, color }: PropsTypes) {
```

Comment:
I think export default is not necessary here, you could go with:
 export const Button = ({ text, onClick, color }: PropsTypes) =>{....
as well, however I don't find it as an error, rather a point where we could use newer ES functionalities.